### DISCUSSION TITLE
permissioin confused me

### DISCUSSION THREAD

**Discussion by JackeyTang** [filebrowser/filebrowser#1952] [General]
if i new a file or drectory,who is the owner?

**Comment by ramiresviana** [comment]
Filebrowser does not handle system permissions, it will just use the privileges of the process owner.

**Reply by JackeyTang** [reply]
thanks. well, when the 'A' user is running the Filebrowser service, another user 'B' want to create file or drectory, then i need to run another Filebrowser service for the user 'B'? is there some good method to solve the problem?

**Reply by ramiresviana** [reply]
You can try using Command Runner (https://filebrowser.org/configuration/command-runner) to change permissions of the files.
